crates/rpc-types-engine/src/attributes.rs

```diff
@@ -60,6 +66,24 @@ impl OpPayloadAttributes {
         self.eip_1559_params.map(decode_eip_1559_params)
     }
 
+    /// Encodes the Jovian `eip1559` parameters for the payload.
+    pub fn get_jovian_extra_data(
+        &self,
+        default_base_fee_params: BaseFeeParams,
```

Comment:
Do we need to edit the `alloy-eips` repo to extend `BaseFeeParams` to hold `minBaseFeeLog2` as well? 

Right now it's always going to passing in the `minBaseFeeLog2` as zero